package-lock.json

```diff
@@ -45633,8 +45633,7 @@
     },
     "react-native-tuvali": {
       "version": "git+ssh://git@github.com/mosip/tuvali.git#768a7fc8b1cd50db97dc5f47d5d70b66ca9d5a3f",
```

Comment:
The checksums for `react-native-tuvali` dependency should have also changed on a version bump up, please run this command as well `npm i react-native-tuvali@github:mosip/tuvali#v0.4.4` and then commit the `package-lock.json` & `Podfile.lock` changes.

Reply:
Added the missed changes also